### DISCUSSION TITLE
Are Typed Lists "officially" supported?

### DISCUSSION THREAD

**Discussion by igoris** [decaporg/decap-cms#7018] [Q&A] [answered]
I can't find any mentions of the typed lists in the docs. However, it seems that they still present in some examples (like here https://github.com/decaporg/decap-cms/blob/master/dev-test/config.yml#L246) and the code processing them is there (https://github.com/decaporg/decap-cms/blob/master/packages/decap-cms-widget-list/src/ListControl.js#L495).

Is this something that can be relied upon or is this a deprecated functionality that will be eventually removed?

**Comment by martinjagodic** [comment, marked as answer]
I think this part of the docs covers this: https://decapcms.org/docs/beta-features/#list-widget-variable-types

This is the PR that delivered the feature: https://github.com/decaporg/decap-cms/pull/1857

There are no plans to deprecate this. If you feel this is not explained enough, a PR for the docs would be great.

**Reply by igoris** [reply]
Oh nice, I must have missed this doc. 
Thank you!
